docs/src/modules/singlelayerqg.md

```diff
@@ -68,8 +68,9 @@ and [`addforcing!`](@ref GeophysicalFlows.SingleLayerQG.addforcing!).
 All required parameters are included inside [`Params`](@ref GeophysicalFlows.SingleLayerQG.Params)
 and all module variables are included inside [`Vars`](@ref GeophysicalFlows.SingleLayerQG.Vars).
 
-For decaying case (no forcing, ``F=0``), `vars` can be constructed with [`DecayingVars`](@ref GeophysicalFlows.SingleLayerQG.DecayingVars). 
-For the forced case (``F \ne 0``) the `vars` struct is with [`ForcedVars`](@ref GeophysicalFlows.SingleLayerQG.ForcedVars) or [`StochasticForcedVars`](@ref GeophysicalFlows.SingleLayerQG.StochasticForcedVars).
+For decaying case (no forcing, ``F = 0``), variables are constructed with [`Vars`](@ref GeophysicalFlows.SingleLayerQG.Vars).
```

Comment:
"For the decaying case..."? This looks to be the same where it appears in subsequent scripts so if you prefer it as is totally fine.